custom/icds_reports/tasks.py

```diff
@@ -70,6 +70,11 @@
     {'type': 'child_list', 'name': 'static-child_health_cases'},
 ]
 
+SQL_FUNCTION_PATHS = [
+    ('migrations', 'sql_templates', 'create_functions.sql'),
+    ('migrations', 'sql_templates', 'database_functions', 'child_health_monthly.sql')
+]
+
```

Comment:
@emord I will move rest of SQL view definitions to the separate files in other PR.